packages/@n8n/nodes-langchain/package.json

```diff
@@ -137,17 +137,17 @@
     "@google-cloud/resource-manager": "5.3.0",
     "@google/generative-ai": "0.11.4",
     "@huggingface/inference": "2.7.0",
-    "@langchain/anthropic": "0.2.9",
+    "@langchain/anthropic": "0.2.15",
     "@langchain/cohere": "0.0.10",
-    "@langchain/community": "0.2.20",
-    "@langchain/core": "0.2.18",
-    "@langchain/google-genai": "0.0.23",
-    "@langchain/google-vertexai": "0.0.21",
-    "@langchain/groq": "0.0.15",
-    "@langchain/mistralai": "0.0.27",
-    "@langchain/ollama": "^0.0.2",
-    "@langchain/openai": "0.2.5",
-    "@langchain/pinecone": "0.0.8",
+    "@langchain/groq": "0.0.17",
+    "@langchain/mistralai": "0.0.28",
+    "@langchain/community": "0.2.28",
+    "@langchain/core": "0.2.27",
+    "@langchain/google-genai": "0.0.26",
+    "@langchain/google-vertexai": "0.0.26",
```

Comment:
can we revert this sorting to be alphabetical again, to make the diff easier to read 🙏🏽

Reply:
Good catch, done!